lib/rex/post/hwbridge/ui/console/command_dispatcher.rb

```diff
@@ -73,7 +73,7 @@ def msf_loaded?
   def log_error(msg)
     print_error(msg)
 
-    elog(msg, 'hwbridge')
+    elog(msg, 'hwbridge', error: $!)
 
     dlog("Call stack:\n#{$@.join("\n")}", 'hwbridge')
```

Comment:
I think we can get rid of the whitesapce and this `dlog` line, now that `elog` will add the stack trace as expected

```suggestion
```